src/test/scala/carldata/series/StatsTest.scala

```diff
@@ -70,4 +70,25 @@ class StatsTest extends FlatSpec with Matchers {
     TimeSeries.almostEqual(resultTs, expected, 0.01) shouldBe true
   }
 
+  it should "allow to find 25 percentile" in {
+    val xs: Seq[Double] = Seq(1, 2, 3, 4)
+    Stats.percentile(xs, 25) shouldBe 1
+  }
+
+  it should "allow to find median" in {
```

Comment:
Could you add one test when the calculated value (e.g. median, has to be estimated?)

Maybe median from (1,2,3,4) = 2.5